### ISSUE TITLE
Support repeated `--disable` options

### ISSUE THREAD

**Issue by kevinrenskers** [nicklockwood/SwiftFormat#396]
I tried to make a `.swiftformat` config file like this:

```
--swiftversion 4.1
--indent 2
--patternlet inline
--disable redundantReturn
--disable yodaConditions
--disable unusedArguments
--disable andOperator
--disable anyObjectProtocol
```

But was surprised to find that this only disabled `anyObjectProtocol` and left the rest enabled. Only when I changed my config file to this:

```
--swiftversion 4.1
--indent 2
--patternlet inline
--disable redundantReturn,yodaConditions,unusedArguments,andOperator,anyObjectProtocol
```

Did it work as expected. This is a lot less readable though, so I was hoping that first way or repeating the `--disable` could be supported?

**Comment by kevinrenskers** [comment]
Seems a bit similar to https://github.com/nicklockwood/SwiftFormat/issues/336, which suggests a YAML format, which would also be nice. Feel free to close this issue as a duplicate :)

**Comment by nicklockwood** [comment]
@kevinrenskers you're right, this should work (or should at least error if it doesn't). I'll get this fixed for the next release.

YAML support is a little further off, but should happen eventually.

**Comment by nicklockwood** [comment]
This feature is now implemented on the [develop](https://github.com/nicklockwood/SwiftFormat/tree/develop) branch.

**Comment by nicklockwood** [comment]
@kevinrenskers fixed in 0.40.4.
